spacetemplate/assets/legacy.yaml

```diff
@@ -87,6 +87,121 @@ work_item_types:
+    order: 3
+    trans_rule_key: "updateStateFromColumnMove"
+    trans_rule_argument: "{ metaState: 'mResolved' }"
+
+- id: "56d62801-798a-4bb0-9c97-89f136f3d539"
+  name: Experiences Board
+  description: This is the default board config for the SDD template (Experiences).
+  context: "8e4c995d-8e5f-4ad6-85bd-3cfbe718f908"
+  context_type: "TypeLevelContext"
+  columns:
+  - id: "8faebb8a-3748-44c6-a691-27633dde571c"
+    name: "New"
+    order: 0
+    trans_rule_key: "updateStateFromColumnMove"
+    trans_rule_argument: "{ metaState: 'mNew' }"
+  - id: "907dad6c-f117-4ad6-b6dd-e21fb198e56d"
+    name: "Approved"
+    order: 1
+    trans_rule_key: "updateStateFromColumnMove"
+    trans_rule_argument: "{ metaState: 'mOpen' }"
+  - id: "90a0a0b1-3e9c-4921-8430-25ff56fd1996"
+    name: "Committed"
+    order: 2
+    trans_rule_key: "updateStateFromColumnMove"
+    trans_rule_argument: "{ metaState: 'mInprogress' }"
+  - id: "86a2aaaa-4a80-433b-b390-b8f42eec2d32"
+    name: "Done"
+    order: 3
+    trans_rule_key: "updateStateFromColumnMove"
+    trans_rule_argument: "{ metaState: 'mResolved' }"
+
+- id: "29493abe-02eb-4e4b-ac3b-a4c1390fa5cd"
+  name: Requirements Board
+  description: This is the default board config for the SDD template (Requirements).
+  context: "8e4c995d-8e5f-4ad6-85bd-3cfbe718f908"
+  context_type: "TypeLevelContext"
+  columns:
+  - id: "6c314706-f562-494d-91b9-b7d2c36672ba"
+    name: "New"
+    order: 0
+    trans_rule_key: "updateStateFromColumnMove"
+    trans_rule_argument: "{ metaState: 'mNew' }"
+  - id: "6b06a763-cdef-400e-98d3-8db46e633c92"
+    name: "Approved"
+    order: 1
+    trans_rule_key: "updateStateFromColumnMove"
+    trans_rule_argument: "{ metaState: 'mOpen' }"
+  - id: "92f48297-062b-4605-9f30-2e546af4d898"
+    name: "Committed"
+    order: 2
+    trans_rule_key: "updateStateFromColumnMove"
+    trans_rule_argument: "{ metaState: 'mInprogress' }"
+  - id: "572c67ef-c550-4084-bd8a-a6d722a3278a"
+    name: "Done"
+    order: 3
+    trans_rule_key: "updateStateFromColumnMove"
+    trans_rule_argument: "{ metaState: 'mResolved' }"
+
+- id: "0e842bef-ac2a-4071-b97a-b07a6b29965d"
+  name: Execution Board
+  description: This is the default board config for the SDD template (Execution).
+  context: "8e4c995d-8e5f-4ad6-85bd-3cfbe718f908"
+  context_type: "TypeLevelContext"
+  columns:
+  - id: "4953fd3a-32dd-4943-8dcf-4b4c9bfcfef1"
+    name: "New"
+    order: 0
+    trans_rule_key: "updateStateFromColumnMove"
+    trans_rule_argument: "{ metaState: 'mNew' }"
+  - id: "eea309e2-8caf-4dc0-98cd-8bb5de3dedb3"
+    name: "Committed"
+    order: 1
+    trans_rule_key: "updateStateFromColumnMove"
+    trans_rule_argument: "{ metaState: 'mOpen' }"
+  - id: "616e8d49-09a9-4ffa-903f-61f215862ee2"
+    name: "In Progress"
+    order: 2
+    trans_rule_key: "updateStateFromColumnMove"
+    trans_rule_argument: "{ metaState: 'mInprogress' }"
+  - id: "c3c4a46e-13d6-4dbb-b82d-bcec22e76275"
+    name: "Completed"
```

Comment:
@michaelkleinhenz I don't see `Completed` and `Verified` state anywhere on slide 19 of https://docs.google.com/presentation/d/18xKQmkf3UKlGoP7qLqUnrIYiflza674n_paUZXKuepA/edit?ts=5afda931#slide=id.g312a66d2fa_0_35. 

Are these the correct values?

Reply:
Yes, see slide 20. Slide 19 describes the meta states, not the columns.